HospitalSystem.Core/People/Doctor/DoctorService.cs

```diff
@@ -28,6 +28,38 @@ public List<Doctor> GetManyBySpecialty(Specialty specialty)
             return specizedDoctors.ToList();
     }
 
+    public List<Doctor> GetManyBySpecialty(string keyword)
+        {
+            var doctors = GetAll();
+            var allDoctors =
+                from doctor in doctors.AsQueryable<Doctor>()
+                select doctor;
+
+            List<Doctor> filteredDoctors = allDoctors.ToList().FindAll(doctor => doctor.Specialty.ToString().Contains(keyword.ToUpper()));
+            return filteredDoctors;
+        }
+
+        public List<Doctor> GetManyByName(string keyword)
+        {
+            var doctors = GetAll();
+            var allDoctors =
+                from doctor in doctors.AsQueryable<Doctor>()
+                select doctor;
+
+            return allDoctors.ToList().FindAll(doctor => doctor.FirstName.Contains(keyword));
+        }
+
+         public List<Doctor> GetManyByLastName(string keyword)
+        {
+            var doctors = GetAll();
+            var allDoctors =
+                from doctor in doctors.AsQueryable<Doctor>()
+                select doctor;
+
+            List<Doctor> filteredDoctors = allDoctors.ToList().FindAll(doctor => doctor.LastName.Contains(keyword));
+            return filteredDoctors;
+        }
```

Comment:
Bad indent WHY ACCEPT YOUR OWN OVER DEV WHEN NO STUFF

Reply:
it has stuff, or maybe its from last pr :D
fixed indent